flake.nix

```diff
@@ -24,11 +23,12 @@
       packages = forAllSystems (system:
         let
           pkgs = nixpkgs.legacyPackages.${system};
+          version = "0.5.3";
```

Comment:
This seems to be misleading. It should rather point to a commitID in combination with a branch name. As a build from "main" is not the same as a installation from the releases tab.

Reply:
Indeed. It looks like the [default template](https://github.com/NixOS/templates/blob/master/go-hello/flake.nix) for Go projects uses:

```nix
    # Generate a user-friendly version number.
      version = builtins.substring 0 8 lastModifiedDate;
```

It's OK if I update to that?

Reply:
Additionally I can leave it as it is and close this pr. The bob version would be 

bob version 0.0.0 linux/amd64 as when we do a bob build